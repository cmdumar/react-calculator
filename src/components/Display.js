import { string } from 'prop-types';

function Display({ result }) {
  return (
    <div>{result}</div>
  );
}

Display.propTypes = {
  result: string.isRequired,
};

export default Display;
