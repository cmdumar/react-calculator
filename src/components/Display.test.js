import React from 'react';
import renderer from 'react-test-renderer';
import Display from './Display';

test('Display Component for displaying the calculations', () => {
  const display = renderer.create(<Display result="222" />).toJSON();

  expect(display).toMatchSnapshot();
});
