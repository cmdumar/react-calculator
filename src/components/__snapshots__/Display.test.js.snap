// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component for displaying the calculations 1`] = `
<div
  className="display"
>
  222
</div>
`;
